src/current/cockroachcloud/tools-page.md

```diff
@@ -144,9 +151,72 @@ To deactivate the integration:
 
 After deactivating an integration, the metrics data will remain in Datadog for a default [retention period](https://docs.datadoghq.com/developers/guide/data-collection-resolution-retention/).
 
+## Monitor CockroachDB {{ site.data.products.cloud }} with Amazon CloudWatch integration
+
+The CockroachDB {{ site.data.products.cloud }} integration for [Amazon CloudWatch]((https://aws.amazon.com/cloudwatch/) enables data collection and alerting on CockroachDB metrics available at the [Prometheus endpoint]({% link {{site.current_cloud_version}}/monitoring-and-alerting.md %}#prometheus-endpoint). It is only available with CockroachDB {{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters hosted on AWS.
+
+{{site.data.alerts.callout_success}}
+Enabling the Amazon CloudWatch integration on your CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster will apply additional charges to your **Amazon CloudWatch** bill. Your CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} bill is unchanged.
+{{site.data.alerts.end}}
+
+### Enable
+
+To enable Amazon CloudWatch monitoring for a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster hosted on AWS, you can either:
+
+- Use the [Cloud API]({% link cockroachcloud/cloud-api.md %}), following the instructions on [Export Metrics From a CockroachDB {{ site.data.products.standard }} Cluster]({% link cockroachcloud/export-metrics.md %}) or [Export Metrics From a CockroachDB {{ site.data.products.advanced }} Cluster]({% link cockroachcloud/export-metrics-advanced.md %}).
+
+OR
+
+- Use the Cloud Console, following the steps below.
+
+1. On the cluster's **Tools** page, click **Setup** in the **CloudWatch** panel.
+
+1. Fill in the **Role ARN**, **Target Region**, and **Log Group Name** fields with their corresponding values.
+    - The **Role ARN** is in the format: `arn:aws:iam::{role_arn}:role/{role_name}`. To determine `{role_arn}` and `{role_name}`, follow steps 1 through 8 on [Export Metrics From a CockroachDB {{ site.data.products.standard }} Cluster]({% link cockroachcloud/export-metrics.md %}#enable-metrics-export) or [Export Metrics From a CockroachDB {{ site.data.products.advanced }} Cluster]({% link cockroachcloud/export-metrics-advanced.md %}#enable-metrics-export).
```

Comment:
Instead of calling out the step numbers, is it possible to refer to a specific subheading or section? If any steps are ever added or removed, this will become stale. I try to avoid referencing step numbers for this reason.